samples/camera/README.rst

```diff
@@ -0,0 +1,30 @@
+.. camera:
+
+Camera Sample Application
+###################################
+
+Overview
+********
+This project demos camera samples.
+It should work with any platform featuring a I2C peripheral interface and a CMOS sensor interface.
+It does not work on QEMU.
+If there is display device on board, the demo displays camera stream on the display device.
+
+Requirements
+************
+This project requires a CMOS sensor connected to board.
```

Comment:
Can you list the sensor (and LCD display panel) you tested this with (so someone could duplicate what you did.

Reply:
Added.